src/services/__test__/qr.service.test.ts

```
@@ -0,0 +1,113 @@
+import { QRLoginTokenRepository } from '@/repositories/qr.repository';
+import { DBError } from '@/exception';
+import { QRLoginToken } from '@/types/models/QRLoginToken.type';
+
+jest.mock('@/repositories/qr.repository');
+
+describe('QRLoginTokenService', () => {
+  let service: QRLoginTokenService;
+  let repo: jest.Mocked<QRLoginTokenRepository>;
+
+  beforeEach(() => {
+    const repoInstance = new QRLoginTokenRepository({} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)